player.lua

```diff
@@ -79,6 +98,13 @@ rightpressed = false
 --- Is the spacebar currently pressed?
 spacepressed = false
 
+--- Is the specified key currently pressed?
+--- @tparam int keycode The specified key that is being checked. A list of available keycodes is at http://www.makeflashgames.com/resources/keycodes.php
```

Comment:
Should link https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/ui/Keyboard.html instead as that's official AS3 documentation.